spec/helper.rb

```diff
@@ -1,2 +1,3 @@
 require 'rspec'
 require 'email_doc'
+require 'active_support/rescuable'
```

Comment:
Because of rspec commands failed. :(

RSpec commands caused failed messages as follows:
```
F

Failures:

  1) EmailDoc::Document#render should render document
     Failure/Error:
       Class.new(ActionMailer::Base) do
         def welcome(email)
           mail \
             to:       email,
             from:     'from@example.org',
             reply_to: 'noreply@example.org',
             subject:  'WELCOME',
             body:     'Welcome to my site.'
         end
       end

     NameError:
       uninitialized constant ActiveSupport::Rescuable
     # ./spec/lib/email_doc/document_spec.rb:5:in `require'
     # ./spec/lib/email_doc/document_spec.rb:5:in `block (2 levels) in <top (required)>'
     # ./spec/lib/email_doc/document_spec.rb:19:in `block (2 levels) in <top (required)>'
     # ./spec/lib/email_doc/document_spec.rb:24:in `block (3 levels) in <top (required)>'
     # ./spec/lib/email_doc/document_spec.rb:37:in `block (4 levels) in <top (required)>'
     # ./spec/lib/email_doc/document_spec.rb:37:in `each'
     # ./spec/lib/email_doc/document_spec.rb:37:in `block (3 levels) in <top (required)>'

Finished in 0.37025 seconds (files took 0.27632 seconds to load)
1 example, 1 failure

Failed examples:

rspec ./spec/lib/email_doc/document_spec.rb:26 # EmailDoc::Document#render should render document
```